client/apollo/react/src/TimelineVertical/TimelineVerticalCommon.tsx

```diff
@@ -0,0 +1,27 @@
+import { PropsWithChildren, ReactNode } from "react";
+import classNames from "classnames";
+
+export type TimelineVerticalProps = PropsWithChildren<{
+  title: string;
+  tag: ReactNode;
+  className?: string;
+}>;
+
+export const TimelineVertical = ({
+  title,
+  children,
+  tag,
+  className,
+}: TimelineVerticalProps) => {
+  return (
+    <section className={classNames("af-timeline-vertical", className)}>
+      <header className="af-timeline-vertical__header">
+        {tag}
```

Comment:
ce ne serait pas plus intéressant de faire ça : 
Cela empêcherait les devs de tout le temps remettre le composant Tag en prop alors que dans tous les cas où ce sera utilisé il y aura besoin de ce composant.

```suggestion
        {tag && <Tag {...tagProps}>{tag}</Tag>}
```

Reply:
déjà le tag est une propriété obligatoire du composant timeline (selon la documentation ui/ux). le challenge vient du fait qu'il y ait une version apollo et une look&feel, du coup satisfaire ces conditions vient avec des compromis

Reply:
Ha en effet, je vais adapter car il est toujours possible d'injecter le composant Tag pour qu'il soit adapté à apollo ou l&f.

```tsx
export const TimelineVertical = ({
  title,
  children,
  tag,
  className,
  TagComponent,
}: TimelineVerticalProps) => (
  <section className={classNames("af-timeline-vertical", className)}>
    <header className="af-timeline-vertical__header">
      <TagComponent>{tag}</TagComponent>
      <h4 className="af-timeline-vertical__title">{title}</h4>
    </header>
    {Boolean(children) && (
      <main className="af-timeline-vertical__description">{children}</main>
    )}
  </section>
);
```

Reply:
similaire à ceci:
> [pourquoi avoir créé un nouvelle prop TagComponent ? il y avait déjà la prop tag, non ?](https://github.com/AxaFrance/design-system/pull/1120#discussion_r2048634569)

Reply:
D'accord, si c'est ça on laisse comme ça alors, je trouve juste dommage de tout le temps devoir mettre le composant Tag alors que ce sera toujours celui là qui sera utilisé 😅